### PR TITLE
Update Materialize@0.100.2 and JQuery@3.2.12

### LINKED ISSUE #160: Not able to assign customizable options to modal
<mz-modal #bottomSheetModal [options]="modalOptions">
  <mz-modal-header>
    Test Header
  </mz-modal-header>
  
</mz-modal>

#### TS file
```typescript
public modalOptions: Materialize.ModalOptions = {
  dismissible: false, // Modal can be dismissed by clicking outside of the modal
  opacity: .5, // Opacity of modal background
  inDuration: 300, // Transition in duration
  outDuration: 200, // Transition out duration
  startingTop: '100%', // Starting top style attribute
  endingTop: '10%', // Ending top style attribute
  ready: (modal, trigger) => { // Callback for Modal open. Modal and trigger parameters available.
    alert('Ready');
    console.log(modal, trigger);
  },
  complete: () => { alert('Closed'); } // Callback for Modal close
};
```

### PR DESCRIPTION
**Pull request by jfcere**
✔️ **Ready to be reviewed/merged**

- [x] Update Materialize to 0.100.x
- [x] Update JQuery to 3.2.x
- [x] Test manually after updates (double check everything)
- [x] Fix unit/view tests
- [x] Fix issues related to updates
  - [x] Remove sidenav fix for mobile resolution (fixCloseOnClick method)
  - [x] Use workaround for bug multiple select options are offset: https://github.com/Dogfalo/materialize/issues/5083#issue-248489869

<hr>
Fixes #160

### CODE REVIEW

**Comment by scote** [comment]
I think we have this Yarn bug on the App Veyor image : https://github.com/yarnpkg/yarn/issues/3313
The current Yarn version is 0.24.3 on Visual Studio 2017 iamge.
They will update it on the next image build : https://github.com/appveyor/ci/issues/1777

**Comment by jfcere** [comment]
Not related to the bug you linked. I reinstalled the same yarn version as AppVeyor and running `yarn install` my lock file updated itself to discover a missing `/`. And voilà...

File: src/app/sidenav/sidenav.component.unit.spec.ts [hunk 1]
```diff
@@ -234,116 +204,4 @@ describe('MzSidenavComponent:unit', () => {
...
-        expect(mockCollapseButton.sideNav).not.toHaveBeenCalled();
-      });
-
-      it('should keep track of listeners', () => {
-
-        expect(component.closeOnClickListeners.length).toBe(1);
-      });
-    });
-
-    describe('collapsible link', () => {
```
Comment (scote) [inline src/app/sidenav/sidenav.component.unit.spec.ts]: Why do we remove those unit tests?

Comment (jfcere) [reply src/app/sidenav/sidenav.component.unit.spec.ts]: The test was meant to be sure that `fixCloseOnClick` won't close the menu on mobile when clicking on a item that should collapse (not a page link)

**Pull request merged**